### DISCUSSION TITLE
Cant preview the uploaded image

### DISCUSSION THREAD

**Discussion by mohamedmoussasoli** [filamentphp/filament#1815] [Help]
### Package

filament/filament

### Package Version

v2.10.21

### Laravel Version

v9.2

### Livewire Version

v2.10.4

### PHP Version

8.1.2

### Bug description

After the new update of filament, i cant preview the uploaded image.
<img width="955" alt="Screenshot 2022-03-06 at 2 15 25 PM" src="https://user-images.githubusercontent.com/9276573/156922688-1e27d789-8d2b-48af-bc96-feb7c48f4fcd.png">

**Comment by danharrin** [comment]
What is the latest Filament version that works correctly for you?

**Comment by mohamedmoussasoli** [comment]
2.0.0

**Comment by danharrin** [comment]
v2.0.0 is not an admin panel version. It started at v2.5.0 I think. Please check the actual version where the preview breaks.

**Comment by mohamedmoussasoli** [comment]
Oh sorry my mistake.

My current version v2.10.21.. and the previous version was v2.10.11

The current version where the preview is not displayed.

**Comment by nabileisa** [comment]
@mohamedmoussasoli I would suggest checking your dev tools ,If there is a `404 (Not Found).` error for the image , you might want to run `php artisan storage:link` to link your storage.

**Comment by saade** [comment]
i'm on the latest and can't confirm that.

Mine is working the way it should

**Comment by danharrin** [comment]
Yeah neither. Unfortunately this is an issue with your environment.

**Comment by jcgmalave** [comment]
In my case, I solved it by modifying the .env of my Laravel app, placing port 8000 in APP_URL=http://localhost:8000
